tooling/template-base/components/LearnerAccess/LearnerAccess.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { AvailableTab, LearnerAccessProps, TabKey } from './Types/types';
+import LoadWaitlist from './LoadWishlistedContent';
+import LoadArchivedContent from './LoadArchivedContent';
+import LoadBookmarks from './LoadBookmarks';
+import LoadUserLearning from './LoadUserLeaning';
+import LoadCertificates from './LoadCertificates';
+import { LoadingDots, useContentGroupsQuery } from '@thoughtindustries/content';
+import LearnerAccessContext from './Context/context';
+import { getAvailableTabs } from './Utilities/utilities';
+import { useTranslation } from 'react-i18next';
+import { localizedTabLabelMapping } from './Constants/constants';
+
+const LearnerAccess = ({
+  classNames,
+  displayExpiredCertificateInformation,
+  query,
+  userHasManagerInterfaceAccess,
+  companyEnableExternalCertificateUploads,
+  companyHasWaitlistingFeature
+}: LearnerAccessProps): JSX.Element => {
+  const [activeTabKey, setActiveTabKey] = useState<TabKey | undefined>(undefined);
+  const [availableTabs, setAvailableTabs] = useState<AvailableTab[]>([]);
+  const [button, setButton] = useState(false);
+  const {
+    loading,
+    error,
+    refetch: refetchContentGroups
+  } = useContentGroupsQuery({
+    variables: {
+      query,
+      includeExpiredCertificates: displayExpiredCertificateInformation
+    },
+    onCompleted: data => {
+      const newAvailableTabs = getAvailableTabs(
+        data.UserContentGroups || [],
+        userHasManagerInterfaceAccess,
+        companyEnableExternalCertificateUploads,
+        companyHasWaitlistingFeature
+      );
+      // reset active tab key if not included in available tabs
+      if (activeTabKey && !newAvailableTabs.find(({ key }) => key === activeTabKey)) {
+        setActiveTabKey(undefined);
+      } else if (!activeTabKey && newAvailableTabs.length) {
+        setActiveTabKey(newAvailableTabs[0].key);
+      }
+      setAvailableTabs(newAvailableTabs);
+    }
+  });
+
+  // update state to display grid only on mobile
+  const handleResize = () => {
+    if (window.innerWidth < 600) {
+      setButton(true);
+    } else {
+      setButton(false);
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('resize', handleResize);
+  }, []);
+
+  const { t } = useTranslation();
+
+  const resetActiveTab = useCallback(
+    () => setActiveTabKey(availableTabs.length ? availableTabs[0].key : undefined),
+    [availableTabs]
+  );
+
+  if (loading) {
+    return <LoadingDots />;
+  }
+
+  if (error) {
+    return <>{error.message}</>;
+  }
+
+  const handleTabSelection = (currentTabKey: TabKey) => {
+    setActiveTabKey(currentTabKey);
+  };
+
+  const styleLi = {
```

Comment:
Know this is pulled from the widget PR so not necessarily yours, but we can get rid of the object structure here as it's not really needed. Something more like

```javascript
const styleLi = clsx(classNames, 'inline-block text-[14px] pt-4 pr-6 relative');
```

Reply:
solution pushed